lib/evm/system.rb

```diff
@@ -5,7 +5,11 @@ def initialize(executable)
     end
 
     def run(*args)
-      Kernel.system(@executable, *args)
+      succeeded = Kernel.system(@executable, *args)
+      unless succeeded
+        print "Failed! See logs above for error."
```

Comment:
Can use please use `puts` here (to get a newline) and use single quotes for consistency.